state/src/db.rs

```diff
@@ -32,13 +32,14 @@ use util_error::UtilError;
 
 use super::{
     Account, ActionHandler, Asset, AssetAddress, AssetScheme, AssetSchemeAddress, Backend, CacheableItem, Metadata,
-    MetadataAddress, Shard, ShardAddress, ShardBackend, TopBackend,
+    MetadataAddress, RegularAccount, RegularAccountAddress, Shard, ShardAddress, ShardBackend, TopBackend,
 };
 
 const STATE_CACHE_BLOCKS: usize = 12;
 
 // The percentage of supplied cache size to go to accounts.
-const ACCOUNT_CACHE_RATIO: usize = 40;
+const ACCOUNT_CACHE_RATIO: usize = 30;
+const REGULAR_ACCOUNT_CACHE_RATIO: usize = 10;
```

Comment:
Isn't this too large?

Reply:
How much is proper do you think?
I set 10% because an account has three fields and a regular account has only one field.
So I set the ratio to 3:1

Reply:
@kseo How about 35:5
If 1/6 users use regular accounts, the size ratio is 35:5